sdp.go

```diff
@@ -315,6 +438,37 @@ func getMidValue(media *sdp.MediaDescription) string {
 	return ""
 }
 
+// TODO(sgotti) add rid parsing
+func getRids(media *sdp.MediaDescription) []string {
+	rids := []string{}
+	for _, attr := range media.Attributes {
+		if attr.Key == "rid" {
+			split := strings.Split(attr.Value, " ")
+			rids = append(rids, split[0])
+		}
+	}
+	return rids
+}
+
+// TODO(sgotti) add simulcast parsing
+func hasSimulcast(media *sdp.MediaDescription) bool {
+	for _, attr := range media.Attributes {
+		if attr.Key == "simulcast" {
+			return true
+		}
+	}
+	return false
+}
+
+func hasExtension(extMaps map[int]*sdp.ExtMap, uri fmt.Stringer) bool {
+	for _, extMap := range extMaps {
+		if extMap.URI.String() == uri.String() {
```

Comment:
I haven't really nitpicked on perf on this review, but in several cases like this, accepting the string URI is clearer IMO